frontend/src/views/Game.tsx

```diff
@@ -409,6 +415,10 @@ function GamePage() {
               onChange={() => ''}
               readOnly
             />
+            <BottomFooterText>
+              {'Notice an issue? Contact us at '}
+              <NoColorExternalLink href="mailto:support@codejoust.co">support@codejoust.co</NoColorExternalLink>
```

Comment:
Check out #119! I've added "copy email" functionality there, I think that could potentially be incorporated into this PR (we can likely merge #119 first since that's a simpler PR, but let me know your thoughts).

Reply:
Sounds good - I'll wait for that PR to merge before implementing it then (since I just approved it as well)

Reply:
I just merged that PR, sounds good 👍 

The "Email copied!" text may overlap with the time in this case, not sure if anything should be done about that (likely not, idk), but just wanted to note that.

Reply:
Done! I moved the copy popup to the bottom to avoid the overlap.

Reply:
Ayyy nice, looks great! I'll note that it has an animation up but then disappears immediately when you close it, but that doesn't actually bother me as I think that looks fine too. (The only note may be a lack of consistency with other pages, but it's already on the bottom of the page instead of the top, so it's not exactly the same anyway.)